p2p_wallet/Scenes/Main/Home/Home.RootView.swift

```diff
@@ -114,6 +139,21 @@ extension Home {
                     balancesOverviewView?.setUp(state: $1, data: $0 ?? [])
                 })
                 .disposed(by: disposeBag)
+
+            viewModel.bannersDriver
+                .drive(onNext: { [weak self] contents in
+                    self?.bannersDataSource.bannersContent = contents
+                    self?.bannersCollectionView.isHidden = contents.isEmpty
```

Comment:
check if bannersCollectionView.isHidden != contents.isEmpty and perform hiding/showing collection view animation